main.js

```diff
@@ -129,9 +126,7 @@ function showWindow() {
   }
 
   // toggle the visibility of the show/hide tray icon menu entries
-  if (tray) {
```

Comment:
tray is initialized after the window is created.  It's possible that the window could attempt to make IPC calls to  interact with the tray before it is initialized causing a crash.  Keeping the null check around the invocation of methods on the tray object is safer.